ykcapi/src/lib.rs

```diff
@@ -238,6 +238,8 @@ pub extern "C" fn yk_stopgap(
     let mut framerec = unsafe { FrameReconstructor::new(activeframes) };
 
     // Parse the stackmap of the JIT module.
+    // YKFIXME: Parsing the stackmap and initialising `framerec` has a 40ms overhead, which could
```

Comment:
Should this be `YKOPT`?

Reply:
Also the 40s metric is specific to yklua. I think it's enough just to say "this is really slow" or something like this.

Reply:
Both fixed in [c8af45b](https://github.com/ykjit/yk/pull/619/commits/c8af45b27186f79fa414ac5cffb345b1fd48077c)